README-dev.md

```diff
@@ -18,9 +13,9 @@ cd up42-py
 make install[dev]
 ```
 
-3. Create a new project on [UP42](https://up42.com).
+2. Create a new project on [UP42](https://up42.com).
 
-4. Create a `config.json` file and fill in the [project credentials](https://docs.up42.com/developers/authentication#step-1-find-project-credentials).
+3. Create a `config.json` file and fill in the [project credentials](https://docs.up42.com/developers/authentication#step-1-find-project-credentials).
```

Comment:
@andher1802 There's a PR that will remove the mentions of project credentials from the DocHub page, see https://up42.atlassian.net/browse/DOC-789. Let's link this page instead: https://sdk.up42.com/authentication/